examples/tutorial/fortran/CMakeLists.txt

```diff
@@ -5,16 +5,20 @@
 # SPDX-License-Identifier: (MIT)
 ##############################################################################
 blt_add_executable(
-  NAME tut_allocator_f
-  SOURCES tut_allocator.f
-  DEPENDS_ON umpire)
-list(APPEND umpire_tutorials tut_allocator_f)
+  NAME tut_allocator_fortran
+  SOURCES tut_allocator.F
+  DEPENDS_ON umpire hip_runtime)
```

Comment:
Can this be run with cuda? Checking this right now...

Reply:
Ah yeah, I think this should be in an in `if (ENABLE_HIP)` block!

Reply:
Yea that won't work.. fixing now. Stay tuned